### PR TITLE
Social Connection admin views

### PR DESCRIPTION
**Pull request by andrewconner**
Added social connections to User view
Added Social User view to see details of specific Social Users
Added Social Users view, listing all social users in the system (fortytwo users and not)
Misc admin view improvements

### CODE REVIEW

File: shoebox/app/views/user.scala.html [hunk 1]
```diff
@@ -22,6 +22,13 @@
             location.href="@com.keepit.controllers.routes.UserController.refreshSocialInfo(userWithSocial.user.id.get)"
           }
         </script>
+        <style>
+          .scrollArea {
```
Comment (eishay) [inline shoebox/app/views/user.scala.html:26]: lets get a decent css file :-) we should do more styling

Comment (andrewconner) [inline shoebox/app/views/user.scala.html:26]: Agreed :)

File: shoebox/app/views/socialUser.scala.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <title>Social User: @socialUserInfo.fullName</title>
+        <link rel="stylesheet" media="screen" href="@routes.Assets.at("stylesheets/main.css")">
+        <link rel="shortcut icon" type="image/png" href="@routes.Assets.at("images/favicon.png")">
+        <script src="@routes.Assets.at("javascripts/jquery-1.7.1.min.js")" type="text/javascript"></script>
+        <script type="text/javascript">
+          function refreshSociaUserInfo() {
```
Comment (eishay) [inline shoebox/app/views/socialUser.scala.html:21]: what's going to happen in this function?!

File: shoebox/app/views/socialUser.scala.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        
+        <h2>User connected to @socialConnections.size Social Users</h2>
+	      <div class="scrollArea">
+  	      @for(socialConnection <- socialConnections) {
+  	        @adminHelper.socialUserDisplay(socialConnection, true)<br/>
+  	      }
+	      </div>
+	      
+	      <br/>
+	      <button onclick="refreshSociaUserInfo()">Refresh Social User Info!</button>
```
Comment (eishay) [inline shoebox/app/views/socialUser.scala.html:65]: does this button work?

File: shoebox/app/views/socialUser.scala.html (hunk 2 above)
Comment (andrewconner) [inline shoebox/app/views/socialUser.scala.html:21]: Whoops, took it out from User and didn't put it back in.

File: shoebox/app/com/keepit/controllers/SocialUserController.scala [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+    Ok(views.html.socialUser(socialUserInfo, socialConnections, rawInfo))
+  }
+
+  def socialUsersView = SecuredAction(false) { implicit request => 
+    val socialUsers = CX.withConnection { implicit c => SocialUserInfo.all.sortWith((a,b) => a.fullName < b.fullName) }
+    Ok(views.html.socialUsers(socialUsers))
+  }
+  
+  def refreshSocialInfo(userId: Id[User]) = SecuredAction(false) { implicit request =>
```
Comment (eishay) [inline shoebox/app/com/keepit/controllers/SocialUserController.scala:55]: will you remove the method from UserController?

**Pull request merged**